modules/Bio/EnsEMBL/Compara/RunnableDB/EpoLowCoverage/LowCoverageGenomeAlignment.pm

```diff
@@ -412,7 +413,11 @@ sub _parse_results {
 			#print "before cigar_line " . $genomic_align->cigar_line . "\n";
 
 			
-			$genomic_align->cigar_line($start_X . "X" .$genomic_align->cigar_line . $end_X . "X");
+			$genomic_align->cigar_line(
+			    Bio::EnsEMBL::Compara::Utils::Cigars::_cigar_element('X', $start_X) .
```

Comment:
I find it unusual for a method to be called like this - my understanding of `_*` methods is that they are internal to the module and would not be called outside of that module. This doesn't follow that convention. Am I too strict with my definition of internal methods?

Reply:
I have factored this pattern out to a new method in Utils::Cigars